src/schema/v2/me/dislikeArtworkMutation.ts

```diff
@@ -0,0 +1,41 @@
+import { GraphQLBoolean, GraphQLNonNull, GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+import { ArtworkType } from "../artwork"
+
+export default mutationWithClientMutationId<any, any, ResolverContext>({
+  name: "DislikeArtwork",
+  description: "Add (or remove) an artwork to (from) a users dislikes.",
+  inputFields: {
+    artworkID: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    remove: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+    },
+  },
+  outputFields: {
+    artwork: {
+      type: ArtworkType,
+      resolve: ({ artwork_id }, _, { artworkLoader }) =>
+        artworkLoader(artwork_id),
+    },
+  },
```

Comment:
This is actually minor but you're gonna want to output `me` as well. Generally speaking you always want to return the thing you're mutating on since we use fragments to update state in the Relay cache.